acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/tls/ClientSideTlsAcceptanceTest.java

```diff
@@ -168,7 +175,7 @@ void ethSignerDoesNotConnectToServerNotSpecifiedInTrustStore(@TempDir Path workD
 
   @Test
   void missingKeyStoreForEthSignerResultsInEthSignerTerminating(@TempDir Path workDir)
-      throws IOException {
+      throws IOException, CertificateException, NoSuchAlgorithmException, KeyStoreException {
```

Comment:
nit: if everyone else is "throws Exception" - should all other tests follow suit (rather than calling out the explicit exceptions?
I don't care either way, but consistent is good

Reply:
yep i think i missed that one